network/dns.go

```diff
@@ -56,7 +56,7 @@ func (c dnsClient) Resolve(host string) ([]net.IP, error) {
 
 	// compute possible names to resolve
 	names := append([]string{}, podDNSConfig.NameList(host)...)
-	names = append([]string{}, nodeDNSConfig.NameList(host)...)
+	names = append(names, nodeDNSConfig.NameList(host)...)
```

Comment:
was this a bug originally?

Reply:
I'm afraid it was, I merged this not so long ago.